dev_tools/devspace.yaml

```
@@ -101,3 +101,10 @@ commands:
 #     tag: v1.0.0
 #   ui:
 #     path: ./ui        # Path-based dependencies (for monorepos)
+
+profiles:
+  - name: go1.22 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)